index.js

```diff
@@ -19,9 +19,14 @@ function main() {
           break;
     
 
-    case 'showAllTasks':
-      taskDisplay.showAllTasks();
-      break;
+        case 'showAllTasks':
+            taskDisplay.showAllTasks();
+            break;
+
+        case 'deleteTask':
+            const deleteTaskTitle = args[1];
+            taskManager.deleteTask(deleteTaskTitle);
+            break;
```

Comment:
Consider using if statements instead of switch case syntax for handling command inputs.
Use simpler command names for a user. Also, the list of available commands can be extended in the future. This also applies to other code in the file.
```suggestion
        case 'show all':
            taskDisplay.showAllTasks();
            break;

        case 'delete':
            const deleteTaskTitle = args[1];
            taskManager.deleteTask(deleteTaskTitle);
            break;
```